### ISSUE TITLE
 C# - Base language support extension, no intellisense in pre-Release v2.0.283

### ISSUE THREAD

**Issue by eric-camplin** [dotnet/vscode-csharp#5957]
Type: <b>Bug</b>

## Issue Description ##
Intellisense stops working after switching to prerelease version of C# extension v2.0.283
## Steps to Reproduce ##
1. Clean install in sandbox of VSCode, .Net7, add C# extensions (release)
1. Open .cs project - Check intellisense
1. switch to prerelease C# extension, restart VSCode
1. Open .cs project - Check intellisense
## Actual Behavior ##
intellisense works for release version of C# extension but not for preview  v2.0.283
### C# log ###
```
Dotnet path: c:\Users\WDAGUtilityAccount\AppData\Roaming\Code\User\globalStorage\ms-dotnettools.vscode-dotnet-runtime\.dotnet\7.0.9\dotnet.exe
Activating C# standalone...
info: LanguageServerHost[0]
      Starting server...
[Info  - 5:29:32 PM] [LanguageServerProjectSystem]Loaded MSBuild at C:\Program Files\dotnet\sdk\7.0.306
[Info  - 5:29:32 PM] [LanguageServerProjectSystem]Loading c:\Users\WDAGUtilityAccount\Desktop\RollDicePlus\RollDice.sln...
```
## Environment information ##

**VSCode version**: 1.80.1
**C# Extension**: 2.0.283
**Using OmniSharp**: false

<details><summary>Dotnet Information</summary>

Host:
  Version:      7.0.9
  Architecture: x64
  Commit:       8e9a17b221

.NET SDKs installed:
  No SDKs were found.

.NET runtimes installed:
  Microsoft.NETCore.App 7.0.9 [c:\Users\WDAGUtilityAccount\AppData\Roaming\Code\User\globalStorage\ms-dotnettools.vscode-dotnet-runtime\.dotnet\7.0.9\shared\Microsoft.NETCore.App]

Other architectures found:
  None

Environment variables:
  Not set

global.json file:
  Not found

Learn more:
  https://aka.ms/dotnet/info

Download .NET:
  https://aka.ms/dotnet/download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|Folder Name|
|---|---|---|---|
|csharp|ms-dotnettools|2.0.283|ms-dotnettools.csharp-2.0.283-win32-x64|
|vscode-dotnet-runtime|ms-dotnettools|1.6.0|ms-dotnettools.vscode-dotnet-runtime-1.6.0|;

</details>

Extension version: 2.0.283
VS Code version: Code 1.80.1 (74f6148eb9ea00507ec113ec51c489d6ffb4b771, 2023-07-12T17:22:07.651Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz (8 x 2995)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|4.00GB (1.76GB free)|
|Process Argv|--crash-reporter-id 19528559-ebcd-4697-a293-8e543babe6a9|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
py29gd2263cf:30792227
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
vscccc:30610679
2e4cg342:30602488
89544117:30613380
showlangstatbar:30737416
vsctsb:30748421
03d35959:30757346
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixshowwlkth:30771522
hideindicator:30785051
pythongtdpath:30769146
i26e3531:30792625
pythonnosmt12:30797651
pythonidxpt:30784022
pythonnoceb:30797650
e537b577:30795824
dsvsc013:30795093
dsvsc014:30797589

```

</details>

**Comment by dibarbet** [comment]
A couple of info requests.  From the logs attached, it does appear as though the project is being attempted to load, but doesn't seem to finish.
1.  Which parts of intellisense aren't working exactly - completion, hover, etc?
2.  Are you able to share the project?
3.  If you can't share the project, could you set the `dotnet.server.trace` setting to `Trace` and attach the C# output window logs?

**Comment by alowe-quest** [comment]
I had the same issue on C# extension version 2.0.320 and 2.0.328. I had not hover information and was unable to go to references. The problem was solved by reverting to version 1.26.0.

**Comment by dibarbet** [comment]
We've made a fair number of fixes to project loading this month (with a couple more on the way).  It is likely this has been resolved so I'm going to close this issue.  If you're still having a problem on the latest, please file a new issue with
1.  The C# output window logs (with dotnet.server.trace set to Trace)
2.  And if you're using devkit, the 'Projects' output window
